yb-voyager/src/srcdb/yugabytedb.go

```diff
@@ -34,6 +34,8 @@ import (
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils/sqlname"
 )
 
+var yugabyteUnsupportedDataTypesForDbzm = []string{"BOX", "CIRCLE", "LINE", "LSEG", "PATH", "PG_LSN", "POINT", "POLYGON", "TSQUERY", "TSVECTOR", "TXID_SNAPSHOT"}
```

Comment:
Shall we add the GEOMETRY, GEOGRAPHY to this list as well? I know we didn't test, but doesn't hurt to put it?

Reply:
Done